sdk/tests/conformance2/buffers/uniform-buffers.html

```diff
@@ -171,7 +171,7 @@
     // Shouldn't be able to bind a deleted buffer.
     gl.bindBuffer(gl.UNIFORM_BUFFER, b2);
     shouldBeNull("gl.getParameter(gl.UNIFORM_BUFFER_BINDING)");
-    wtu.glErrorShouldBe(gl, gl.NO_ERROR, "there should be no errors");
+    wtu.glErrorShouldBe(gl, gl.INVALID_OPERATION, "there should be INVALID_OPERATION");
```

Comment:
Can you move this one line above? Also, change the message to "binding a deleted buffer generates INVALID_OPERATION"?